index.js

```diff
@@ -17,6 +17,25 @@ var offsets = {
     sizes = { s: 12, m: 18, l: 24 },
     makiRenders = __dirname + '/renders/';
 
+var aliases = {
+    'america-football': 'american-football',
+    'camera': 'attraction',
+    'chemist': 'college',
```

Comment:
I wasn't happy with the ambiguity of the "chemist" icon, [dug deeper and found](https://phabricator.wikimedia.org/T300040#7707095) that it was most certainly meant to represent a pharmacy.